logic/src/commonMain/kotlin/com/wire/kalium/logic/data/asset/AssetMapper.kt

```diff
@@ -46,4 +50,32 @@ class AssetMapperImpl : AssetMapper {
             downloadedDate = Clock.System.now().toEpochMilliseconds()
         )
     }
+
+    override fun fromMessageEntityToAssetProtoContent(messageEntity: MessageEntity): AssetProtoContent {
+        // TODO: Complete mapping once all the necessary extra fields for the Asset Message have been added to the DB
+        return AssetProtoContent(
+            original = AssetProtoContent.Original(
+                mimeType = "*/*", // TODO: add a mimeType to the MessageEntity table
+                size = 0, // TODO: add a size field to the MessageEntity table
+                name = "", // TODO: map the asset name from the exif info or the original file title?
+                metadata = getAssetMessageMetadata(null),
+                source = null, // TODO: add a source field to the MessageEntity DB table
+                caption = null // TODO: add a caption field to the MessageEntity DB table
+            ),
+            preview = null,
+            uploadStatus = null
+        )
+    }
+
+    private fun getAssetMessageMetadata(mimeType: String?): AssetProtoContent.AssetMetadata {
+       return when {
+            mimeType?.contains("image/") ?: false -> Image(width = 0, height = 0, tag = null)
```

Comment:
I was wondering if this could be simplified since we are just returning an `Image` `AssetMetadata` on all cases, or add like a `AssetMetadata.Unknown` type to return in case not mapping could be made on the else block.
 
Still, I think this is WIP, so if you think it isn't necessary to address it now, we can do it on the upcoming PRs if you agree

Reply:
Ahh yeah sorry, this will be overriden or removed by the following PR, so don't pay too much attention to it. But thanks for the catch anyways 🦅